src/modules/analytics/collections-analytics.service.ts

```diff
@@ -64,10 +63,11 @@ export class CollectionsAnalyticsService {
   public async getCollectionsOrderByVolum(
     limit: number = 10,
     offset: number = 0,
+    series: string = '',
```

Comment:
maybe use null or undefined for default value

Reply:
done